GoogleDataTransport/GDTCORLibrary/GDTCORFlatFileStorage.m

```diff
@@ -450,8 +450,8 @@ - (void)checkForExpirations {
 
           // Decode the expired event from the path to be deleted.
           NSError *decodeError;
-          GDTCOREvent *event = (GDTCOREvent *)GDTCORDecodeArchive([GDTCOREvent class], pathToDelete,
-                                                                  nil, &decodeError);
+          GDTCOREvent *event = (GDTCOREvent *)GDTCORDecodeArchiveAtPath([GDTCOREvent class],
+                                                                        pathToDelete, &decodeError);
           if (event == nil || decodeError) {
```

Comment:
The same reasoning applies here as well. This was the only location in the source where the changed API was used.